tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import csv
+import os
+from collections import defaultdict
+from itertools import islice
+from typing import Any, DefaultDict, Dict, Generator, List, Tuple
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import CategoryInfo, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.label.basic import AttributeType
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import NameList, chunked
+
+_DATASET_NAME = "VGGFace2"
+_SEGMENTS = ("train", "test")
+
+
+_CLASS_ATRRIBUTES = ("Sample_num", "Flag", "Gender")
```

Comment:
what is the `CLASS` in `_CLASS_ATTRIBUTE` mean?
And the `ATRRIBUTES` is misspelled.
Can we extract the attribute names from the catalog?